src/Runner/Parallel/SerializedClosureTask.php

```diff
@@ -0,0 +1,64 @@
+<?php
+declare(strict_types=1);
+
+namespace GrumPHP\Runner\Parallel;
+
+use Amp\Cancellation;
+use Amp\Parallel\Worker\Task;
+use Amp\Sync\Channel;
+use Laravel\SerializableClosure\SerializableClosure;
+
+/**
+ * @template-covariant TResult
+ * @template TReceive
+ * @template TSend
+ *
+ * @implements Task<TResult, TReceive, TSend>
+ */
+class SerializedClosureTask implements Task
```

Comment:
This task is rather straight forward, you might want to reconsider supporting it in `parallel-functions` for v3 anyways. For now, it's here to stay.